src/components/ErrorMessage.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FunctionComponent, ReactNode } from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+
+import "lbh-frontend/lbh/components/lbh-error-message/_error-message.scss";
+
+export type ErrorMessageProps = {
+  id?: string;
+  className?: string;
+  visuallyHiddenText?: string;
+  children?: ReactNode;
```

Comment:
I think, in practice, `children` is always added to the props type you pass in (have a look at the `FunctionComponent` interface declaration to see why). I think it's good to document it explicitly, but just a heads up.